x/config/chooser_test.go

```diff
@@ -381,11 +384,35 @@ func TestChooserConfigurator(t *testing.T) {
 						unary:
 							fake-transport:
 								bogus-list:
-									fake-updater: fake-news
+									fake-updater: {}
 			`),
 			wantErr: []string{
 				`failed to configure unary outbound for "their-service": `,
-				`no recognized peer list in config: got bogus-list; need one of fake-list, least-pending, round-robin`,
+				`no recognized peer list "bogus-list"`,
+				`need one of`,
+				`fake-list`,
+				`least-pending`,
+				`round-robin`,
```

Comment:
Given that peerListNames always sorts the list, can we do an exact match on "need one of"?